libs/sdk-ui-dashboard/src/model/commands/filters.ts

```diff
@@ -472,6 +520,7 @@ export function changeAttributeFilterSelection(
     filterLocalId: string,
     elements: IAttributeElements,
     selectionType: AttributeFilterSelectionType,
+    title?: string,
```

Comment:
This is problematic and I would  consider it as a "breaking change", as the command is already public. But does it make sense at all to include title in this command? It's mainly about changing the selection? What is the use case to include title here?